cluster/src/test/resources/node2conf/iotdb-engine.properties

```diff
@@ -0,0 +1,29 @@
+#Licensed to the Apache Software Foundation (ASF) under one
+#or more contributor license agreements.  See the NOTICE file
+#distributed with this work for additional information
+#regarding copyright ownership.  The ASF licenses this file
+#to you under the Apache License, Version 2.0 (the
+#"License"); you may not use this file except in compliance
+#with the License.  You may obtain a copy of the License at
+#
+#http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing,
+#software distributed under the License is distributed on an
+#"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#KIND, either express or implied.  See the License for the
+#specific language governing permissions and limitations
+#under the License.
+
+
+base_dir=target/tmp2
```

Comment:
how about manage these directories just like `target/node1, target/node2, target/node3`?,and then the parameters just like `base_dir` 
 in `node1conf/iotdb-engine.properties` can be `base_dir=target/node1/tmp`.I think it's better so that I can enter `target/node1` and all sub directories in this directory will belongs to node1?

Reply:
thanks, have changed, the three nodes will have different folders to hold the data.